### PR TITLE
chore(views): replace inline html with elgg_view_module when applicable

### PR DESCRIPTION
**Pull request by jdalsem**
kept all the HTML and inline JS intact... that deserves another ticket

There is also a js bug in the notifications plugin (not caused by these changes). I will report a seperate issue for this.

### CODE REVIEW

File: mod/notifications/views/default/forms/notificationsettings/groupsave.php [hunk 1]
```diff
@@ -27,85 +27,78 @@
...
+				'class' => "{$method}toggleOff",
+				'border' => '0',
+				'onclick' => "adjust{$method}_alt('{$method}{$group->guid}');",
+			]);
+			
+			$fields .= elgg_format_element('td', ['class' => "{$method}togglefield"], $toggle_link);
 			$i++;
 		}
+		
+		$group_row = elgg_format_element('td', ['class' => 'namefield'], elgg_format_element('div', [], $group->name));
```
Comment (mrclay) [inline mod/notifications/views/default/forms/notificationsettings/groupsave.php]: `"<td class='namefield'><div>{$group->name}</div></td>"` would be so much more readable. I think we should generally avoid using `elgg_format_element` unless it really cleans up mess or attributes are variable. Maybe @hypeJunction has another perspective here.

The last time I looked, Moodle pages were full of function-generated elements and it made reading difficult.

File: mod/logbrowser/views/default/logbrowser/refine.php [hunk 2]
```diff
@@ -5,36 +5,26 @@
...
+$module_options = ['id' => 'log-browser-search-form'];
 if (!isset($vars['user_guid']) && !isset($vars['username'])) {
-	$form_class .= ' hidden';
+	$module_options['class'] = 'hidden';
 }
 
-?>
+$module = elgg_view_module('inline', elgg_echo('logbrowser:search'), $form, $module_options);
 
-<div id="logbrowser-search-area" class="mbm">
```
Comment (mrclay) [inline mod/logbrowser/views/default/logbrowser/refine.php]:
```php
<div id="logbrowser-search-area" class="mbm">
    <?= $toggle_link . $module ?>
</div>
```

Besides readability, PHPStorm can also see what's going on, recognize ids/classes in our stylesheets, etc.

File: mod/notifications/views/default/forms/notificationsettings/groupsave.php (hunk 1 above)
Comment (hypeJunction) [inline mod/notifications/views/default/forms/notificationsettings/groupsave.php]: I prefer to inline html tags outside of php. 
https://github.com/hypeJunction/Elgg-user_settings/blob/master/views/default/notifications/subscriptions/rows/groups.php

Comment (hypeJunction) [inline mod/notifications/views/default/forms/notificationsettings/groupsave.php]: In this particular case, I don't have any preference, as it is messy no matter what you do with it. In general, I prefer to create a separate view for module body, as it ends up cleaner and allows me to use inline tags.

File: mod/notifications/views/default/notifications/subscriptions/forminternals.php [hunk 3]
```diff
@@ -6,40 +6,30 @@
  */
```
Comment (mrclay) [inline mod/notifications/views/default/notifications/subscriptions/forminternals.php]: This view desperately needs breaking into subviews. I'd rather we did that first (far easier to verify correctness).

Comment (jdalsem) [inline mod/notifications/views/default/notifications/subscriptions/forminternals.php]: it is easier after this cleanup than before....

**Comment by mrclay** [comment]
The big portion of this that simply moves module HTML to `elgg_view_module` is great.

We should resist the urge to convert readable HTML structures to string-building code. Where we need HTML in variables (`elgg_view_module`) we should use `<<<` strings, output buffering, or introduce subviews.

I don't know if we formally adopted `<?= $short ?>` PHP template tags, but we should embrace them. A good decision PHP made was making those always supported.

File: mod/notifications/views/default/notifications/subscriptions/forminternals.php (hunk 3 above)
Comment (mrclay) [inline mod/notifications/views/default/notifications/subscriptions/forminternals.php]: Agreed on this one. Existing code is a total mess. We'll just have to test this manually very carefully.

Comment (jdalsem) [inline mod/notifications/views/default/notifications/subscriptions/forminternals.php]: There already is an existing ticket to split this form... https://github.com/Elgg/Elgg/issues/4434

**Comment by jdalsem** [comment]
i reduced the 'excessive' amount of elgg_format_element usage... i think this can go in as is

**Pull request merged**